less/home-page.less

```diff
@@ -1,50 +1,147 @@
 // Home Content
+header {
+   
+  background-color: @nav-bg;
+  border: @nav-border;
+  font-style: @nav-font;
+  
 
-.home {
+  @media @phone {
+    .wide
+  }
+
+  
+  
 
+h2 {
+  .flexer;
+  .middle;
+}
+nav {
+  
+  .flexer;
+  .middle;
+   padding: 2%;
+   
+   
+   
+        
+   }
+   li {
+    
+     display: inline;
+     align-items: space-around;
+     padding: 10px;
+     
+        
+   }
+
+ 
+  } 
```

Comment:
These 40 lines above are what I was mentioning in the index page. Less is more in programming and those extra tags, added styles, and depth complexity that made it more difficult to achieve the desired style.

Reply:
``` CSS
.navbar{
    display: flex;
    flex-direction: column;
    text-align:center;
    justify-content: center;
    background-color:@nav-bg;
    margin: 0 auto;
    
    h2{
        font-size: 3.2rem;
        margin: 1vh auto;
    }
    
    nav{
        display: flex;
        justify-content: center;
        background: @nav-bg;
        margin:1vh auto;
          
        
        a{
            text-decoration: none;
            color: @nav-font;
            font-size:1.3rem;
            margin: 0 1.5vw; 
        }
    }
}
```

Would be the CSS for the First replacement example I commented on line 23.